src/main/java/com/izmaylov/module2/service/ShopService.java

```diff
@@ -0,0 +1,80 @@
+package com.izmaylov.module2.service;
+
+import com.izmaylov.module2.exception.InvalidString;
+import com.izmaylov.module2.model.Customer;
+import com.izmaylov.module2.model.Invoice;
+import com.izmaylov.module2.model.Product;
+import com.izmaylov.module2.util.Util;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class ShopService {
+    private static final int limit = Util.createLimit();
+    private final Invoice invoice = new Invoice();
+    private final ArrayList<Product> products = invoice.getProducts();
+    private final Map<Customer, ArrayList<Product>> ledger = new LinkedHashMap<>();
+    private int orderCount = 0;
+
+
+    public Map<Customer, ArrayList<Product>> getLedger() {
+        return ledger;
+    }
+
+
+
+    public void createOrder(Customer customer) {
+        ArrayList<Product> order = new ArrayList<>();
+        int productsQuantity = Util.randomNumber(5,1);
+        for (int i = 0; i < productsQuantity; i++) {
+            if (products.get(i).isEmptyValue()) {
+                try {
+                    throw new InvalidString();
+                } catch (InvalidString e) {
+                    e.getMessage();
+                }
+            } else {
+                order.add(products.get(i));
+            }
+        }
```

Comment:
Products from the range 1-5 will always be taken, and not random in the amount of 1-5